spec/requests/api/v3/courses/index_spec.rb

```diff
@@ -191,9 +191,9 @@
           json_response = JSON.parse(response.body)
           course_ids << json_response.dig("data", 0, "id")
 
-          expect {
-            get page_3
-          }.to raise_error(Pagy::OverflowError)
+          get page_3
+
+          expect(response).to have_http_status(:bad_request)
 
           expect(course_ids.compact).to eql([course_b.id.to_s, course_a.id.to_s])
```

Comment:
are we still expected to return course ids after bad request?

Reply:
This was a failing spec that seemed to assert the `Pagy::Overflow` error would be raised in this scenario. I'm not entirely sure of the context but it looks like it's dealing with course_ids for the previous successful requests (not the third). Perhaps this assertion for `Pagy::Oveflow` is redundant here as I've already added a spec for it in the new public api for courses.

Reply:
Update: I've reverted this scenario back to it's original assertion based on this pr https://github.com/DFE-Digital/teacher-training-api/commit/475bf2781629d3b04247a8ae32fed36de50f7488#diff-230985fb50f1d3b01a7b55d41484ee88R201